File: task-03/haskell/primen.hs
isPrime :: Int -> Bool
isPrime n
    | n <= 1    = False
    | otherwise = all (\x -> n `mod` x /= 0) [2..(n - 1)]

main :: IO ()
main = do
    putStrLn "Enter a number: "
    input <- getLine
    let x = read input :: Int
    putStrLn "Prime numbers:"
    mapM_ (\i -> if isPrime i then putStrLn (show i ++ " : prime") else return ()) [1..x]

File: task-03/c++/primen.cpp
#include <iostream>

int main(){
    int n;
    std::cout << "enter a no.  :";
    std::cin>>n;
    for(int i=1;i<n+1;i++){
        int f=0;
        for(int j=2;j<i;j++){
            if(i%j==0){
                f=1;
                break;
            }
        }
        if(f==0){
            std::cout <<i<<" : Prime"<<std::endl;
        }
    }
    return 0;
}
